### DISCUSSION TITLE
Failed on install WDK.vsix while creating Windows2022 image from powershell

### DISCUSSION THREAD

**Discussion by manuelmainz** [actions/virtual-environments#6524] [General questions]
Hallo everyone,

I am trying to setup a BuildPipeline with the provided powershell script. Unfortunately the script fails while installing some packages.
The command I use to start the Powershell script.
`GenerateResourcesAndImage -SubscriptionId {0123-abcd} -ResourceGroupName "SoftwareDeploy" -ImageGenerationRepositoryRoot "$pwd" -ImageType Windows2022 -AzureLocation "West Europe"`.

That's the error I get:

> ==> vhd: Provisioning with powershell script: C:\runner-images\images\win/scripts/Installers/Install-WDK.ps1
    vhd: Downloading wdksetup.exe...
    vhd: Downloading package from: https://go.microsoft.com/fwlink/?linkid=2196230 to path C:\Users\packer\AppData\Local\Temp\wdksetup.exe .
    vhd: Package downloaded successfully in 0.47 seconds
    vhd: Starting Install wdksetup.exe...
    vhd: Installation successful in 131.1 seconds
    vhd: Starting Install WDK.vsix...
    vhd: Unsuccessful exit code returned by the installation process: 2003.
==> vhd: Script exited with non-zero exit status: 1. Allowed exit codes are: [0]
==> vhd: Step "StepProvision" failed

Can anyone give me a hint what might cause this issue?

I also tried to remove this package form the json, but then I get similar errors at other packages like "Install-Vsix", "Install-Stack",...

Thanks in advance.

**Comment by djh82uk** [comment]
I've also been getting this, along with other random failures, It's either this, other vsix failures, WIX or WDK.  Not had a successful build for weeks.  Struggling to track down the issue.

**Reply by manuelmainz** [reply]
Yeah, a lot of random errors on a lot of different packages. I am back to using a local server with a self hosted agent to keep deployment running.
I am wondering if this script even works for someone out of the box?

**Comment by mikhailkoliada** [comment]
It is a generic network hiccup, we have been observing it for a while, too, then it got disappeared
